webpack.config.js

```diff
@@ -1,40 +1,42 @@
-const CleanWebpackPlugin = require('clean-webpack-plugin')
+const CleanWebpackPlugin = require("clean-webpack-plugin").CleanWebpackPlugin;
```

Comment:
See https://github.com/johnagan/clean-webpack-plugin/releases/tag/v3.0.0 for why we have to do this